packages/app/src/cli/models/extensions/specifications/app_config_webhook_subscription.ts

```diff
@@ -0,0 +1,78 @@
+import {WebhookSimplifyConfig} from './app_config_webhook.js'
+import {WebhooksConfig} from './types/app_config_webhook.js'
+import {WebhooksSchema} from './app_config_webhook_schemas/webhooks_schema.js'
+import {CustomTransformationConfig, createConfigExtensionSpecification} from '../specification.js'
+import {getPathValue} from '@shopify/cli-kit/common/object'
+
+export const WebhookSubscriptionSpecIdentifier = 'webhook_subscription'
+
+type TransformedWebhookSubscription = {
+  api_version: string
+  uri: string
+  topic: string
+  compliance_topics?: string[]
+  sub_topic?: string
+  include_fields?: string[]
+}
+
+function transformFromWebhookSubscriptionConfig(content: object) {
```

Comment:
I remember us getting quite confused here around what the returned object would look like. do you think it's worthwhile to add a return type here? and for the other transform function?